recipe/meta.yaml

```diff
@@ -69,3 +68,4 @@ extra:
     - moorepants
     - bollwyvl
     - AakashGfude
+    - mmcky
```

Comment:
@mmcky please post a comment as to whether you're interested in helping maintain this package

Reply:
I'm going to pull this here so we can ship, please do a follow-on PR/rerender